pipelines/batch/src/main/java/com/google/fhir/analytics/ConvertResourceFn.java

```diff
@@ -145,6 +145,9 @@ public void writeResource(HapiRowDescriptor element)
     if (!parquetFile.isEmpty()) {
       startTime = System.currentTimeMillis();
       parquetUtil.write(resource);
+      if (createParquetViews) {
```

Comment:
With the suggestion that I have made below in `ParquetUtil` we don't need this `if` block either.

Reply:
Done.